examples/dummy_plugin/dummy_plugin/__init__.py

```diff
@@ -2,14 +2,14 @@
 
 
 class DummyPluginConfig(PluginConfig):
-    name = "nautobot.extras.tests.dummy_plugin"
+    name = "dummy_plugin"
     verbose_name = "Dummy plugin"
     version = "0.0"
     description = "For testing purposes only"
     base_url = "dummy-plugin"
     min_version = "0.9"
     max_version = "9.0"
-    middleware = ["nautobot.extras.tests.dummy_plugin.middleware.DummyMiddleware"]
+    middleware = ["dummy_plugin.middleware.DummyMiddleware"]
     installed_apps = ["nautobot.extras.tests.dummy_plugin_dependency"]
```

Comment:
Is the installed_app going to get moved to `examples` as well?

Reply:
I am on the fence about the dummy dependency moving.